go-kit/metrics/provider/librato/uniquecounter_test.go

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2018 Salesforce
```

Comment:
Re: filename ... This does not test the uniquecounter it tests `HLLCounter`.

Reply:
Fair enough.